barcodes_generator_partner/__init__.py

```diff
@@ -1,3 +1,5 @@
+# Copyright (C) 2019-Today: La Louve (<https://cooplalouve.fr>)
```

Comment:
No need to add La Louve in modules we are migrating for the OCA
Just Druidoo here